cs_api_io/cs_api_io.ml

```diff
@@ -77,3 +77,33 @@ let get_response {Response.code; body} =
   else
     let message = Printf.sprintf "HTTP response: %d\n%s" code body in
     Lwt_result.fail message
+
+let get_graphql_errors (json : Yojson.Safe.t) =
+  (match json with
+  | `Assoc _ -> (Some (Yojson.Safe.Util.member "errors" json), [])
+  | _ -> (None, ["Unexpected response from the server"]))
+  |> function
+  | (None, errors) -> errors
+  | (Some `Null, _) -> []
+  | (Some errors_json, _) -> (
+    match errors_json with
+    | `List error_list ->
+      error_list
+      |> CCList.map (Yojson.Safe.Util.member "message")
+      |> CCList.map (function
+           | `String error_message -> error_message
+           | _ -> "Unexpected response from the server")
+    | _ -> ["Unexpected response from the server"])
+
+let get_response_parse_error {Response.code; body} =
```

Comment:
What about calling this `get_response_graphql` or something similar to clarify that this function should only be used for GraphQL responses?

Reply:
Good idea.